src/test/rgw/sfs/test_rgw_sfs_sfs_bucket.cc

```diff
@@ -223,6 +227,9 @@ TEST_F(TestSFSBucket, UserCreateBucketCheckGotFromCreate) {
   }
 
   EXPECT_EQ(bucket_from_create->get_acl(), arg_aclp);
+  EXPECT_TRUE(
+      bucket_from_create->get_modification_time().time_since_epoch().count() > 0
```

Comment:
I think `real_clock::is_zero(ts)` is the canonical way to check that the timestamp is not zero.
`EXPECT_NE(real_clock::is_zero(bucket_from_create->get_modification_time()))`